samples/image-create-and-upload-sample/image_create_and_upload_script.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+source config.sh
+ 
+function standardize_object_name() {
+  log "Formatting Object name $1"
+  local object_original_url=$1
+  local object_original_name=${object_original_url##*/}
+  local object_extension=""
+  local object_temp_name=""
+  if echo $object_original_name | grep -q -i rhcos; then
+    echo $object_original_name | grep 'qcow2.gz' >/dev/null
+    [ $? -ne 0 ] && error "Unsupported file format"
+    object_extension="ova.gz"
+    object_temp_name=${object_original_name%.*.*}
+  elif echo $object_original_name | grep -q -i rhel; then
+    echo $object_original_name | grep 'qcow2' >/dev/null
+    [ $? -ne 0 ] && error "Unsupported file format"
+    object_extension="ova.gz"
+    object_temp_name=${object_original_name%.*}
+  fi
+  IMAGE_NAME=$(echo $object_temp_name | sed -e 's/\.//g' -e 's/ppc64le//g' -e 's/openstack//g' -e 's/kvm//g' -e 's/---*//g' -e 's/_/-/g'|grep -o -E '[a-z]*-[0-9]{2}')-$(date +%m%d%Y)
+  OBJECT_NAME=$IMAGE_NAME.${object_extension}
```

Comment:
It will be great if you can add a comment here with some description

Reply:
Added description